docs/toc.md

```diff
@@ -25,11 +24,51 @@
 ## [Authoring Libraries for .NET Core RC2](scenarios/library-authoring/index.md)
 ### [How to manage package dependency versions for .NET Core RC2](scenarios/library-authoring/fastening-libraries-to-rc2.md)
 ### [How to trim your package dependencies](scenarios/library-authoring/trimming.md)
+# [.NET Concepts](concepts/index.md)
+## [What is the .NET Core Platform?](concepts/what-is-dotnet.md)
+## [The .NET Primer](concepts/primer.md)
+## [.NET Standard](concepts/dotnet-standard-library.md)
+## [Frameworks](frameworks.md)
+## [Overview of .NET implementations](concepts/editions-overview.md) 
+## [Common Language Runtime (CLR)](essentials/clr.md)
+## [Class Library overview](essentials/classlibrary.md)
+## [Working with base types](essentials/basetypes.md)
+## [Exception management in .NET Core](essentials/exceptions.md)
+## [🔧 Introduction to metadata in the .NET Platform](essentials/metadata.md)
+## [🔧 Automatic memory management and garbage collection](essentials/gc.md)
+## [Programming Languages on .NET Platform](languages/index.md)
+### [🔧 .NET - a multi-language platform](languages/overview.md)
+### [🔧 The main trio: C#, F# and Visual Basic](languages/main-trio.md)
+## [Understanding package management on .NET](packaging/index.md)
+### [🔧 Introducing NuGet](packaging/overview.md)
+### [🔧 Working with NuGet packages](packaging/working-with-nuget-packages.md)
+### [🔧 Creating NuGet packages](packaging/creating-nuget-packages.md)
+### [🔧 Publishing NuGet packages](packaging/publishing-nuget-packages.md)
+### [Collections and Data Structures](essentials/collections/Collections-and-Data-Structures.md)
```

Comment:
Starting here, up to my Line comment <end> is a lot of more in-depth than everything else in this list.

I think it should move down, or even be a link to 'resources about the .NET standard library'.